Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Mantrap.xml

```diff
@@ -20,14 +20,14 @@
 			<li Class="PatchOperationReplace">
 				<xpath>/Defs/ThingDef[defName="AA_Mantrap"]/statBases/ArmorRating_Blunt</xpath>
 				<value>
-					<ArmorRating_Blunt>1.5</ArmorRating_Blunt>
+					<ArmorRating_Blunt>10</ArmorRating_Blunt>
 				</value>
 			</li>
 			
 			<li Class="PatchOperationAdd">
 				<xpath>/Defs/ThingDef[defName="AA_Mantrap"]/statBases</xpath>
 				<value>
-					<ArmorRating_Sharp>0.5</ArmorRating_Sharp>
+					<ArmorRating_Sharp>2</ArmorRating_Sharp>

```

Comment:
No. In fact, feel free to remove the armor from this thing entirely. It's a damned _plant_.